apps/frontend/src/pages/Project/Builds.tsx

```diff
@@ -49,7 +49,7 @@ const ProjectQuery = graphql(`
         id
         url
       }
-      buildNames
+      buildNames(recentOnly: true)
```

Comment:
Is it used elsewhere without recentOnly? I don't think so, you can probably remove the parameter and only update the resolver?

Reply:
It's used in the automation

Reply:
Why would you like to automate a month past build? 🧐